doc-content/enterprise-only/openshift/environment-managed-deploy-proc.adoc

```diff
@@ -6,12 +6,7 @@
 :modifylink: environment-managed-modify-proc
 = Deploying a managed environment
 
-Deploy a managed environment using one of the following template files:
-
-* `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-sit.yaml` if you need to use minimal resources and do not need high-availability configuration for {LOCAL_CENTRAL} and Smart Router
-* `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod.yaml` if you need high-availability configuration for {LOCAL_CENTRAL} and Smart Router
-+
-You can extract the template files from the `{PRODUCT_FILE}-openshift-templates.zip` product deliverable file. You can download the file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page.
+Deploy a managed environment using the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod.yaml` template file. You can extract the file from the `{PRODUCT_FILE}-openshift-templates.zip` product deliverable file. You can download this file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page.
```

Comment:
This is OK, but Stetson came up with a format that we use consistently that I think works very well, that you can adapt. However, these are procedure steps and shouldn't be in the context statement. Either have these in another proc, and make these prereqs (You have a deployed a managed OS environement).
Or, add these a the first steps to this proc. But I had a look at proc, and really, you really really need to break this up into several different procs. I'm not against nexted steps, but you are doing too much in this one module. The test for me is, how easy is it to write the into? And in this case you have not provided an intro to this procedure, saying what it does a and why to do it. So I suggest having a good look at this - if not now then as a follow up. Let me know if you would like to talk though ideas

Reply:
We can discuss when we're both online. Unfortunately I do not see how this can be broken into several procedures, because it is actually a single atomic procedure - "deploy a template". All the steps are just setting many parameters in the same template; you do it all in one oc command or one screen.

Reply:
Not fixing in this task as this is an issue in all documents. We can have a meeting about both the download format part and the idea of splitting the task.